canonical-web-and-design/maas-ui
# Upgrade legacy vanilla

## Done
- Updated legacy vanilla from v2.0.1 to v2.10.1
- Made footer sticky for react and angular apps
- Fixed a few styling things as drivebys:
  - Header strips now follow same format as react machine list (disable take action with tooltip unless some selected, text next to title = "1 of 2 devices selected" etc)
  - KVM details > Configuration tags input
  - KVM details > Compose form interface selection
  - Device details > Edit interface form table

## QA
- Check that the footer is sticky on the react and angular apps
- Click everything and navigate everywhere, there should hopefully be no styling regressions. Thankfully there are nowhere near as many changes as vanilla 1.8 or 2.0

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1998
Fixes canonical-web-and-design/MAAS-squad#2000
